android/src/main/java/org/th317erd/react/DynamicFontsModule.java

```
@@ -152,7 +155,10 @@ public void loadFont(final ReadableMap options, final Callback callback) throws
       cacheFile.delete();
     } catch(Exception e) {
       callback.invoke(e.getMessage());
-    } finally {
```

can't we just remove this finally and move invoke to the end of the try block rather then creating this extra invoked variable where we have to manage the state in a lot of places. This seems like the only path that would cause double invocations. 
Also have you been able to reproduce and debug? I feel like sentry logs might be hiding the real exception that is occurring.

---

Unfortunately, I wasn't able to reproduce that